### PR TITLE
Add PTH-460 configuration

### LINKED ISSUE #891: Add Support for Wacom PTH-460
Does not get detected yet

[Diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/6175434/Diagnostics.txt)

  "DevicePath": "\\\\?\\hid#vid_056a&pid_0392&mi_00#8&38875c26&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}",
  "VendorID": 1386,
  "ProductID": 914,
  "CanOpen": true,
  "ReleaseNumberBcd": 265,
  "ReleaseNumber": "1.9",
  "ProductVersion": 265,
  "Manufacturer": "Wacom Co.,Ltd.",
  "ProductName": "Wacom Intuos Pro S",
  "SerialNumber": "0LQ0111002200",
  "MaxInputReportLength": 192,
  "MaxOutputReportLength": 0,
  "MaxFeatureReportLength": 2561

### PR DESCRIPTION
**Pull request by X9VoiD**
## Verification

[Discord](https://discord.com/channels/615607687467761684/615607687467761690/792379318428499969)

Fixes #891

### CODE REVIEW

**Comment by AbstractQbit** [comment]
Wacom driver doesn't let you do SetFeature (at least that's the case with pth860), so the second identifier will probably fail if used.

**Comment by X9VoiD** [comment]
#599 will help with that. Can't remove it since this scenario may happen.

- Wacom driver, and OTD installed
- User disabled wacom service or killed process (At this point, tablet is already initialized by wacom driver)
- User, at some point, replugged the tablet (Replugging uninitializes the tablet)

If OTD don't have the SetFeature for the second identifier then the tablet won't be initialized.

**Comment by AbstractQbit** [comment]
It won't be initialized either way because wacom driver doesn't let set feature through 🤷

**Comment by InfinityGhost** [comment]
I never checked if this was ready, is it ready for merge?

**Comment by X9VoiD** [comment]
Yes, ready for merge

File: OpenTabletDriver/Configurations/Wacom/PTH-460.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      ]
+    }
+  ],
+  "AuxilaryDeviceIdentifiers": [
+    {
+      "VendorID": 1386,
+      "ProductID": 914,
+      "InputReportLength": 44,
+      "OutputReportLength": null,
+      "ReportParser": null,
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/Wacom/PTH-460.json:59]: Explicitly define the report parser.

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

File: OpenTabletDriver/Configurations/Wacom/PTH-460.json (hunk 1 above)
Comment (X9VoiD) [reply OpenTabletDriver/Configurations/Wacom/PTH-460.json]: Fixed

**Comment by NBruderman** [comment]
Any news about this one? I think it's ready

**Comment by InfinityGhost** [comment]
> Any news about this one? I think it's ready

Yeah I forgot to re-review. Will give it a once over.

File: OpenTabletDriver/Configurations/Wacom/PTH-460.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      "ProductID": 914,
+      "InputReportLength": 44,
+      "OutputReportLength": null,
+      "ReportParser": "OpenTabletDriver.Vendors.Wacom.WacomDriverIntuosV3ReportParser",
+      "FeatureInitReport": null,
+      "OutputInitReport": null,
+      "DeviceStrings": {},
+      "InitializationStrings": []
+    }
+  ],
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/Wacom/PTH-460.json:65]: Kind of odd to see that this only uses the Wacom driver report parser. Does this still work correctly when caught up to v0.5.2.2?

Comment (X9VoiD) [reply OpenTabletDriver/Configurations/Wacom/PTH-460.json:65]: I'm not entirely sure with this either. This auxiliary endpoint is not the one reporting tablet buttons so we don't know what this endpoint is for. Can change to non-wacom driver if that's more preferable.

Comment (AbstractQbit) [reply OpenTabletDriver/Configurations/Wacom/PTH-460.json:65]: 44 byte endpoint on intuosv3 carries touch data.

Comment (X9VoiD) [reply OpenTabletDriver/Configurations/Wacom/PTH-460.json:65]: Is it confirmed that we need to skip a byte for the 44 byte endpoint?

**Review by InfinityGhost — APPROVED** [review]
lg2m

**Pull request merged**